shell/platform/android/io/flutter/plugin/platform/PlatformViewWrapper.java

```diff
@@ -109,6 +141,10 @@ public void setTexture(@Nullable SurfaceTexture newTx) {
       } else {
         canvas.drawColor(Color.TRANSPARENT);
       }
+
+      if (Build.VERSION.SDK_INT == Build.VERSION_CODES.Q) {
```

Comment:
nit: we decided that it's more readable to use a number than the constant. e.g.
consider: `Build.VERSION_CODES.Q` vs `29`.  Fewer characters, and also more clear to someone reading this code.